src/stack-hci/azext_stack_hci/aaz/latest/stack_hci/arc_setting/_consent_and_install_default_extension.py

```diff
@@ -0,0 +1,276 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "stack-hci arc-setting consent-and-install-default-extension",
```

Comment:
This command signature is too long. Could we simplify it to `stack-hci arc-setting install-default-extension`? @priyjain358 What do you think of it?
```suggestion
    "stack-hci arc-setting install-default-extension",
```

Reply:
@zhoxing-ms we can't change this command signature.

Reply:
@priyjain358 May I ask what is the reason?

Reply:
just install-default-extension doesn't capture the whole intent behind the function.

Reply:
OK, got it